tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, run_in_tmpdir, rustc};
+
+// Attempt to build this dependency tree:
+//
+//	A.1   A.2
+//	 |\    |
+//	 | \   |
+//        B  \  C
+//         \ | /
+//          \|/
+//           D
+//
+// Note that A.1 and A.2 are crates with the same name.
+
+fn main() {
+    run_in_tmpdir(|| {
+        rustc().metadata("1").extra_filename("-1").input("a.rs").run();
+        rustc().metadata("2").extra_filename("-2").input("a.rs").run();
+        rustc().input("b.rs").extern_("a", "liba-1.rlib").run();
+        rustc().input("c.rs").extern_("a", "liba-2.rlib").run();
+        println!("before");
```

Comment:
I guess that we don't need to port the `echo`s, those are just leftover debugging statements I suppose :)

Reply:
they exist to make the test output readable in case it fails, otherwise you will just get an error about d.rs not being able to be compiled, and won't know which instance caused the error.

Reply:
Ok, fair enough, let's keep them then.